src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -42,7 +47,7 @@
     "bigscience/bloom-1b3",
     "bigscience/bloom-2b5",
     "bigscience/bloom-6b3",
-    "bigscience/bloom-176b",
+    "bigscience/bloom",
```

Comment:
I'm not sure which one will be the final name, cc @younesbelkada

Reply:
Indeed the final model is `bloom`and not `bloom-176b` ! Thanks for spotting the nit :)